index.html

```diff
@@ -42,7 +42,159 @@ <h1>I'm Denzel Glad to see you!</h1>
                 <span><a href="#"><i class="fab fa-angellist"></i></a></span>
             </div>
         </section>
+        <section class="projects_section">
+            <div class="project">
+                <div class="image_div">
```

Comment:
Kindly add more padding/margin to center a little bit more your navbar logo.

Fix padding/margin for the navbar logo, as the Figma design you selected. 

![image](https://user-images.githubusercontent.com/60164239/129413470-bf174114-3161-4a3c-9c20-780af1c6db9f.png)